Task_1.py

```diff
@@ -0,0 +1,160 @@
+import os
+from source import get_data
+import csv
+from operator import itemgetter
+
+
+# 1) Task_1.py: to create file general-stats.cvs containing information about:
+#    1.1. average rating (overall) of each application (asin);
+def avg_rating(main_dict):
+    asins = []
+    for item in main_dict.values():
+        asins.append(item['asin'])
+    asins_value = dict.fromkeys(set(asins), [0, 0])
+    for item in main_dict.values():
+        asins_value[item['asin']] = [asins_value.get(item['asin'])[0] + item['overall'], asins_value.get(item['asin'])[1]+1]
+    avg_overall = []
+    for key, value in asins_value.items():
```

Comment:
Please, use [list-comprehensive](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions) it will keep the code more concise.